src/components/GobanThemePicker/GobanThemePicker.tsx

```diff
@@ -118,11 +123,15 @@ export class GobanThemePicker extends React.PureComponent<GobanThemePickerProper
                             className={"selector" + (this.state.board === theme.theme_name ? " active" : "")}
                             style={{
                                 ...theme.styles,
-                                ...(theme.theme_name === "Plain" ? {backgroundColor: boardCustom} : {})
+                                ...(theme.theme_name === "Plain" ? {backgroundImage: "linear-gradient(-45deg, orange,yellow,green,blue,indigo,violet)"} : {})
                             }}
                             onClick={this.selectTheme["board"][theme.theme_name]}
                             >
-                            <PersistentElement elt={this.canvases.board[idx]} />
+
+                            {theme.theme_name === "Plain"
+                                ? <span>CUSTOM</span>
```

Comment:
Everything's gotta be translatable, for this I would use something like

`<span>{pgettext("Custom board theme", "CUSTOM")}</span>`

and also make sure the span has some css applied that makes that text either hides the overflow (`overflow: hidden`) or wraps (`word-wrap: break-word`), whichever you think looks better

Reply:
Yup, I actually did both, wrapped the text and hidden any overflow on the off chance the translation was that long. 

I think both the custom selector and the default text in the form should now be translatable. Thanks for checking it out!